autogen/runtime_logging.py

```diff
@@ -20,11 +20,27 @@
 is_logging = False
 
 
-def start(logger_type: str = "sqlite", config: Optional[Dict[str, Any]] = None) -> str:
+def start(
+    logger: Optional[BaseLogger] = None,
```

Comment:
@jackgerrits, do you think we want to provide using an `logger` instance as an alternative way to start logging? Or should we register the logger class and still use the `logger_type` and `config` to create a logger ourselves?

Reply:
Got it. I think we can retire the factory in a separate PR. 

We don't yet have a user guide documentation for logging. Once we do we want to document the API as only using `logger` as input. Also update this notebook as well: https://microsoft.github.io/autogen/docs/notebooks/agentchat_logging

Reply:
sure @ekzhu , once the test modules are created, ill update the notebook.